### DISCUSSION TITLE
Call to undefined function powerGridJsFramework() [Upgrading from v1 to v2]

### DISCUSSION THREAD

**Discussion by zaimazhar** [Power-Components/livewire-powergrid#312] [Q&A]
These are the steps I have taken to upgrade the library;
- Updating `composer.json`
  - `"power-components/livewire-powergrid": "^1.4"` to `"power-components/livewire-powergrid": "^2.0"`
- Delete `composer.lock` and `vendor`
- `composer install`
- Following this [guide](https://livewire-powergrid.com/#/get-started/upgrade-v1) (I chose to use CDN)
- Start server

![Screenshot from 2022-03-01 01-00-16](https://user-images.githubusercontent.com/54124001/156025607-d18f4b1e-35ad-4eb4-8311-a1872db318cb.png)

Please correct me if I am doing the upgrading process wrong.

Thanks.

**Comment by dansysanalyst** [comment]
You probably have the `v1` views published.

Try running:

```shell
php artisan vendor:publish --tag=livewire-powergrid-views
````

and maybe also:

```shell
php artisan view:clear
```

**Reply by zaimazhar** [reply]
Thank you for the response, however the problem still persists.

I removed the package from the `composer.json` and deleted some other files;
- `vendor`
- `composer-lock.json`
- `config/livewire-powergrid.php`
- `resources/views/vendor/livewire-powergrid/*`

Then, reinstall it using the install [guide](https://livewire-powergrid.com/#/get-started/install).

Now it works perfectly with the latest version and no breaking changes.

![Screenshot from 2022-03-02 11-52-25](https://user-images.githubusercontent.com/54124001/156291750-0f44b930-46b2-4354-9cc2-031d1588cf3c.png)
